server/server.js

```diff
@@ -2,27 +2,29 @@ const express = require('express');
 const path = require('path');
 const bodyParser = require('body-parser');
 const cookieParser = require('cookie-parser');
+const quizerController = require('./controllers/quizController')
 
 const app = express();
+app.use(express.json());
 
 app.use('/build', express.static(path.resolve(__dirname, '../build')))
 
 app.get('/', (req,res)=> {
     //get request to login/ signup page
-    return res.status(200).sendFile(path.resolve(__dirname,'../index.html'))
+    return res.status(200).
+    sendFile(path.resolve(__dirname,'../index.html'))
 })
 
-app.get('/quizoverflow', (req,res)=>{
+app.get('/quizoverflow', quizerController.getQuestion, (req,res)=>{
     //dummy function, placeholder for information to be sent back to client
```

Comment:
Take out this comment to avoid confusion when looking back at this now not dummy function.

Reply:
I've resolved this and removed comment.